neurips2022/track2/train/utility_IL.py

```diff
@@ -86,11 +91,15 @@ def load_data_scratch(dataset_path, save_path):
                 last_sim_time = image_names[-1].split("_Agent")[0]
```

Comment:
1) Try to avoid using "Agent" to get the timestamp.
2) Only use the file naming format, shown below, as the only information available in order to extract the timestamp.
    ![image](https://user-images.githubusercontent.com/24203326/189713402-252e4d17-38cd-4996-8d16-2395369120f3.png)